specs/notation-cli.md

```diff
@@ -0,0 +1,233 @@
+# Notation CLI
+
+The following spec outlines the notation CLI.
+The CLI commands are what's currently available in [notation v0.7.1-alpha.1](https://github.com/notaryproject/notation/releases/tag/v0.7.1-alpha.1)
+
+## Table of Contents
+- [notation](#notation)
+- [sign](#sign): Signs artifacts
+- [verify](#verify): Verifies OCI Artifacts
```

Comment:
nit: Sign command just says "Sign artifacts", and verifies says "Verifies OCI artifacts". Is that on purpose.

Reply:
i just copy pasted what the cli currently has. So, yeah, bug that should be changed. 
This PR is what exists to be modified.